app/src/view/screens/Cinematics/Cinematics.js

```
@@ -152,15 +148,12 @@ export default function Cinematics({ route }) {
 
     try {
-      const response = await fetch(
-        base_url + "cinematics/getTopVideo",
-        {
-          method: "GET",
-          headers: {
-            Authorization: `Bearer ${token}`,
-          },
-        }
-      );
+      const response = await fetch(base_url + "cinematics/getTopVideo", {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)